dspace-api/src/main/java/org/dspace/app/requestitem/RequestItemEmailNotifier.java

```diff
@@ -0,0 +1,227 @@
+                    List<Bundle> bundles = item.getBundles("ORIGINAL");
+                    for (Bundle bundle : bundles) {
+                        List<Bitstream> bitstreams = bundle.getBitstreams();
+                        for (Bitstream bitstream : bitstreams) {
+                            if (!bitstream.getFormat(context).isInternal() &&
+                                    requestItemService.isRestricted(context,
+                                    bitstream)) {
+                                email.addAttachment(bitstreamService.retrieve(context,
+                                        bitstream), bitstream.getName(),
+                                        bitstream.getFormat(context).getMIMEType());
+                            }
+                        }
+                    }
+                } else {
+                    Bitstream bitstream = ri.getBitstream();
+                    email.addAttachment(bitstreamService.retrieve(context, bitstream),
+                            bitstream.getName(),
+                            bitstream.getFormat(context).getMIMEType());
+                }
+                email.send();
+            } catch (MessagingException | IOException | SQLException | AuthorizeException e) {
+                LOG.warn(LogHelper.getHeader(context,
+                        "error_mailing_requestItem", e.getMessage()));
+                throw new IOException("Reply not sent:  " + e.getMessage());
+            }
+        }
+        LOG.info(LogHelper.getHeader(context,
+                "sent_attach_requestItem", "token={}"), ri.getToken());
+    }
+
+    /**
+     * Send, to a repository administrator, a request to open access to a
+     * requested object.
+     *
+     * @param context current DSpace session
+     * @param ri the item request that the approver is handling
+     * @throws IOException if the message body cannot be loaded or the message
+     *          cannot be sent.
+     */
+    static public void requestOpenAccess(Context context, RequestItem ri)
+            throws IOException {
+        Email message = Email.getEmail(I18nUtil.getEmailFilename(context.getCurrentLocale(),
+                "request_item.admin"));
+
+        // Which Bitstream(s) requested?
+        Bitstream bitstream = ri.getBitstream();
+        String bitstreamName;
+        if (bitstream != null) {
+            bitstreamName = bitstream.getName();
+        } else {
+            bitstreamName = "all"; // TODO localize
+        }
+
+        // Which Item?
+        Item item = ri.getItem();
+
+        // Fill the message's placeholders.
+        message.addArgument(bitstreamName);     // {0} bitstream name or "all"
+        message.addArgument(item.getHandle());  // {1} Item handle
+        message.addArgument(ri.getToken());     // {2} Request token
+        message.addArgument(ri.getReqName());   // {3} Requester's name
+        message.addArgument(ri.getReqEmail());  // {4} Requester's address
```

Comment:
I could be mistaken, but isn't it a bit odd to have the the request for open access coming from the person who submitted the Item Request?  It seems like this request should be from the *author* (who checked the box in the form) to the submitter or site admin?